.github/workflows/notify_repositories.yaml

```
@@ -0,0 +1,39 @@
+name: Notify repositories of changes in definitions
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "definitions/**"
+
+jobs:
+  notify-repositories:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repository: ${{ fromJson(vars.NOTIFY_REPOS) }}
+
+    steps:
+      - name: Send repository_dispatch to ${{ matrix.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        run: |
+          TAG="${{ github.event.release.tag_name }}"
+          REPO="${{ matrix.repository }}"
+          echo "Notifying repository: $REPO"
+          curl --version
+          STATUS_CODE=$(curl -s -w "%{http_code}" -o response.txt -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${GITHUB_TOKEN}" \
+            https://api.github.com/repos/${REPO}/dispatches \
+            -d '{"event_type":"update-admin-api-definitions","client_payload":{"tag":"'"${TAG}"'"}}'
```

The payload structure should match the push event context for staging deployments (OPS-293). The [GitHub API documentation](https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event) allows up to 10 top-level properties in client_payload.
```suggestion
            -d '{"event_type":"update-admin-api-definitions","client_payload":{"sha":"'"${COMMIT_SHA}"'","ref":"${{ github.ref }}","branch":"${{ github.ref_name }}","repository":"${{ github.repository }}"}}
```